docs/syntax/links.md

```diff
@@ -16,6 +16,17 @@ A link contains link text (the visible text) and a link destination (the URI tha
 
 [**Hi**, _I'm md_](links.md)
 
+## Link to another page
+
+You can link to another page by pointing to the markdown file.
+This applies to files within the same repo.
+
+```markdown
+[Link text](/solutions/search.md)
+```
+
+Note that they must be prefixed with `/`.
```

Comment:
Links to other pages don't have to be prefixed with `/`. You can also link to other pages with relative links. For example:

```
[syntax guide](../syntax/index.md)
```

and

```
* [Transition from AsciiDoc to Markdown](./syntax.md)
```

Reply:
I think you could update the example in `## Inline link` section to include a relative and absolute link. The example of `links.md` is a bad one and should be replaced.

Reply:
yeah agreed I'll open a fresh PR and try to clean up the whole page tomorrow

Reply:
@bmorelli25 just for the absolute syntax, was operating off this statement https://elastic.slack.com/archives/C07APH4RCDT/p1738594924973629?thread_ts=1738594418.206959&cid=C07APH4RCDT